Gemfile

```diff
@@ -12,7 +12,12 @@ end
 
 gemspec
 
-gem 'rake'
+if RUBY_VERSION < '1.9.3'
```

Comment:
I also think that this deprecation should be in a separate commit rather than tied to mock changes.